### PR TITLE
chore: add sql examples

### PR DESCRIPTION
**Pull request by morenol**

### CODE REVIEW

**Review by sehz — APPROVED** [review]
👍

File: primitives/sql/cli/dataflow.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        type: i32
+      speed:
+        type: float32
+
+topics:
+  vehicle-sensor:
+    schema:
+      value:
+        type: vehicle-data-type
+  vehicle-check:
```
Comment (crajcan) [inline primitives/sql/cli/dataflow.yaml]: Is this topic needed?

**Review by crajcan — APPROVED** [review]

File: primitives/sql/join/README.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
+# SQL (simple query)
+
+In this example, we'll count the occurrences of words in two topics. We will use the CLI interface to run some SQL queries and then use a JOIN query on a service to find the number of occurrences of a word in both topics.
```
Comment (ajhunyady) [inline primitives/sql/join/README.md:3]: We should also summarize word-probe here.

**Review by ajhunyady — APPROVED** [review]
Awesome job @morenol! 
Only feedback is a small nit.

File: primitives/sql/cli/dataflow.yaml (hunk 1 above)
Comment (morenol) [reply primitives/sql/cli/dataflow.yaml]: Not really

**Pull request merged**